### ISSUE TITLE
Ghost markers when updating series data with accessibility & styled mode & disabled markers

### ISSUE THREAD

**Issue by ddragula** [highcharts/highcharts#24164]
#### Expected behaviour
After updating the series' dataset, no markers are visible when marker.enabled: false.

#### Actual behaviour
After updating the series' dataset with accessibility module on & styled mode on, disabled (ghost) markers are shown. (Related to https://github.com/highcharts/highcharts/issues/23878)

#### Live demo with steps to reproduce
https://jsfiddle.net/BlackLabel/ef7uLghy/
Click the `Set Dataset` button.

<img width="866" height="429" alt="Image" src="https://github.com/user-attachments/assets/328f7c71-b468-42cd-bcae-b47c2319b227" />

#### Product version
Highcharts Core v12.5.0

**Comment by magdalena-gut** [comment]
Hi @ddragula , thanks for reporting this issue! I’ve added it to our backlog for further investigation
